control/operation_mode_transition_manager/launch/operation_mode_transition_manager.launch.xml

```diff
@@ -1,6 +1,11 @@
 <launch>
   <arg name="operation_mode_transition_manager_param" default="$(find-pkg-share operation_mode_transition_manager)/config/operation_mode_transition_manager.param.yaml"/>
+  <!-- <arg name="operation_mode_transition_manager_param" default="$(find-pkg-share autoware_launch)/config/control/operation_mode_transition_manager/operation_mode_transition_manager.param.yaml"/> -->
   <arg name="vehicle_info_param_file" default="$(find-pkg-share vehicle_info_util)/config/vehicle_info.param.yaml"/>
+  <arg name="nearest_search_param_path" default="$(find-pkg-share autoware_launch)/config/control/common/nearest_search.param.yaml"/>
+
+  <arg name="vehicle_model" default="sample_vehicle"/>
+  <arg name="vehicle_param_file" default="$(find-pkg-share $(var vehicle_model)_description)/config/vehicle_info.param.yaml"/>
```

Comment:
Currently, the parameter in ```vehicle_param_file```is not loaded into operation_mode_transion_manager. 
Is this what you expected?